scripts/utils.py

```diff
@@ -0,0 +1,96 @@
+from typing import Dict
+from typing import Union
+
+from sample_size.sample_size_calculator import SampleSizeCalculator
+
+DEFAULT_ALPHA = 0.05
+
+
+def is_float(value: str) -> float:
+    try:
+        float(value)
+        return True
+    except ValueError:
+        return False
+
+
+def get_raw_input(text: str) -> str:
+    return input(text)
+
+
+def get_input(input_name: str) -> float:
+    input_str = get_raw_input(f"Enter the {input_name}: ")
+    if is_float(input_str):
+        return float(input_str)
+    else:
+        raise Exception(f"Error: Please enter a float for the {input_name}.")
+
+
+def get_alpha() -> Union[float, None]:
+    alpha_default_check = get_raw_input("Do you want to use default alpha (0.05) for the power analysis? (y/n)")
+    if alpha_default_check.lower() == "n":
+        alpha = get_input("alpha")
+        if 0 <= alpha <= 0.3:
+            return alpha
+        else:
+            raise Exception("Error: Please provide a float between 0 and 0.3 for alpha.")
+    else:
```

Comment:
how about making it into just one question? e.g
```
Enter the alpha between (between 0 and 0.3 inclusively) or press enter to use default alpha=0.05
```
then checking input equals empty string to set alpha = 0.05, otherwise throw error?